docs/doxygen/images/defender_demo.pu

```diff
@@ -0,0 +1,83 @@
+@startuml
+autonumber
+
+box "Application" #LightGreen
+    participant "Demo" as Demo
+    participant "Metrics Collector" as Metrics
+    participant "Report Builder" as Report
+end box
+
+box "Libraries" #LightBlue
```

Comment:
There is no mention of the AWS IoT broker in the sequence diagram. The Shadow demo diagram does contain the broker as part of the workflow. 
https://github.com/aws/aws-iot-device-sdk-embedded-C/blob/master/docs/doxygen/images/shadow_demo_main.pu#L20-L22
For consistency with other diagrams and for better reader's understanding about the "big" picture of the demo, adding the broker participant in the diagram would be good imo

Reply:
Good suggestion. Done.